packages/bpk-docs/src/pages/NativeCardPage/NativeCardPage.js

```diff
@@ -0,0 +1,99 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import readme from './../../../../../native/packages/react-native-bpk-component-card/readme.md';
+
+import iosScreenshotDefault from './../../../../../native/packages/react-native-bpk-component-card/screenshots/ios/default.png';
+import androidScreenshotDefault from './../../../../../native/packages/react-native-bpk-component-card/screenshots/android/default.png';
+import iosScreenshotWithoutPadding from './../../../../../native/packages/react-native-bpk-component-card/screenshots/ios/without-padding.png';
+import androidScreenshotWithoutPadding from './../../../../../native/packages/react-native-bpk-component-card/screenshots/android/without-padding.png';
+
+
+import Paragraph from './../../components/Paragraph';
+import DocsPageBuilder from './../../components/DocsPageBuilder';
+
+
+const components = [
+  {
+    id: 'default',
+    title: 'Default',
+    blurb: 'The default card comes with padding already applied to get you up and running quickly and has normal and hover states baked in.',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotDefault}`,
+        altText: 'iOS Default Card Component',
+        subText: '(iPhone 6 Simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotDefault}`,
+        altText: 'Android Default Card Component',
+        subText: '(Nexus 5 Emulator)',
+      },
+    ],
+  },
+  {
+    id: 'without-padding',
+    title: 'Without Padding',
+    blurb: 'For full flexibility you can opt to remove the default padding e.g. to display full-bleed images.',
+    screenshots: [
+      {
+        title: 'iOS',
+        width: 750,
+        height: 1334,
+        src: `/${iosScreenshotWithoutPadding}`,
+        altText: 'iOS Card Without Padding Component',
+        subText: '(iPhone 6 Simulator)',
+      },
+      {
+        title: 'Android',
+        width: 1080,
+        height: 1920,
+        src: `/${androidScreenshotWithoutPadding}`,
+        altText: 'Android Card Without Padding Component',
+        subText: '(Nexus 5 Emulator)',
+      },
+    ],
+  },
+];
+
+const NativeTextPage = () => <DocsPageBuilder
+  title="Cards"
+  blurb={[
+    <Paragraph>
+      Cards are used to group related items within the UI.
+      They allow complex datasets to be broken down into individual, distinct areas for easy consumption.
+      Cards are used as a signifier to give affordance to a clickable set of elements.
+    </Paragraph>,
+    <Paragraph>
+      There are four types of card: primary, secondary, destructive and featured. All card types can be
+      large, selected or disabled.
```

Comment:
Should this paragraph be here - isn't that just for buttons?

Reply:
yep, nice one, thanks!